### ISSUE TITLE
Integrated terminal always eats up tab key

### ISSUE THREAD

**Issue by isidorn** [Microsoft/vscode#8342]
#8222

1. It is not possible to tab out of the terminal which can make it hard for keyboard accessibilty
2. This also makes the actions in the terminal header not accessible by keyboard

The editor has a command to use tab key for moving focus, maybe something similar is needed for the terminal.

**Comment by Tyriar** [comment]
Duplicate of https://github.com/Microsoft/vscode/issues/6731
